.travis.yml

```diff
@@ -28,7 +28,8 @@ script:
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors_detail
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_arp_table
-- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_peers
+- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_stats
+#- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_stats
```

Comment:
Because I will implement get_ntp_peers to return a list of NTP peers as configured on the device.